truss/base/trt_llm_config.py

```diff
@@ -90,6 +90,9 @@ class TrussTRTQuantizationConfiguration(PydanticTrTBaseModel):
     calib_size: int = 1024
     calib_dataset: str = "cnn_dailymail"
     calib_max_seq_length: int = 2048
+    max_gpu_usage_fraction: float = 0.8
+    attention_implementation: Optional[str] = None
+    pad_tokenizer_during_calibration: Optional[bool] = False
```

Comment:
pad_tokenizer_during_calibration can be infered from the huggingface tokenizer config, right?

Reply:
like the padding side, if not left could cause some severe quality issues: https://huggingface.co/unsloth/Llama-3.2-1B-Instruct/blob/main/tokenizer_config.json#L2063